docs/content/en/docs/getting-started/overview.md

```diff
@@ -23,10 +23,10 @@ EKS Anywhere offers two cluster deployment topology options:
 This type of cluster contains all Cluster API (CAPI) management components needed to manage itself, including managing its own upgrades.
```

Comment:
I'm concerned about this page when you take out the provider-specific Overviews. The remaining content is not good.

I'm going to take a stab at what I think this page needs to be and will push it up here